### PR TITLE
[feat](test) add fe ut for bf index create table

### PR DESCRIPTION
**Pull request by airborne12**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:
1. Fix wrong gram_size 256 for ngram bf index DDL.
2. Enhance bloom_filter_columns empty properties prcocessing in bf index DDL.
3. Add FE UT for ngram and bf index DDL.

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [x] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by airborne12** [comment]
run buildall

**Comment by airborne12** [comment]
run buildall

**Comment by airborne12** [comment]
run buildall

**Comment by airborne12** [comment]
run buildall

**Review by eldenmoon — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java [hunk 1]
```diff
@@ -256,21 +261,29 @@ public void checkColumn(Column column, KeysType keysType, boolean enableUniqueKe
...
-                        throw new AnalysisException("gram_size should be integer and less than 256");
-                    }
-                    if (bfSize > 65535 || bfSize < 64) {
-                        throw new AnalysisException("bf_size should be integer and between 64 and 65535");
-                    }
-                } catch (NumberFormatException e) {
-                    throw new AnalysisException("invalid ngram properties:" + e.getMessage(), e);
-                }
+
+                parseAndValidateProperty(properties, NGRAM_SIZE_KEY, MIN_NGRAM_SIZE, MAX_NGRAM_SIZE);
```
Comment (csun5285) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java:265]: need to modify `IndexDefinition.java`

**Review by csun5285 — CHANGES_REQUESTED** [review]

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**